src/iotjs_exception.c

```diff
@@ -22,7 +22,7 @@
 #include "uv.h"
 
 
-iotjs_jval_t iotjs_create_uv_exception(int errorno, const char* syscall) {
+iotjs_jval_t iotjs_create_uv_exception(ssize_t errorno, const char* syscall) {
```

Comment:
Why do we have to change type to `ssize_t` here? I think here `int` is enough because the function `uv_strerror` using `errorno` accepts `int`, and this amount of range will be fixed(Currently 76) at some point. Even though it is run on how many bit machine. the number itself won't increase.
```c
const char* uv_strerror(int err) {
```